target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -184,6 +185,11 @@ ramips_setup_macs()
 	wiznet,wizfi630s)
 		wan_mac=$(macaddr_add "$(mtd_get_mac_binary factory 0x28)" 1)
 		;;
+	wavlink,wl-wn577a2)
+		wan_mac=$(mtd_get_mac_binary factory 0x2e)
+		lan_mac=$(mtd_get_mac_binary factory 0x28)
```

Comment:
You already have the LAN address set in DTS, so you can drop it here.

Reply:
ACK